test/e2e/tests/add-account.spec.js

```diff
@@ -157,12 +157,11 @@ describe('Add account', function () {
         });
 
         // Check address of 2nd account
-        const accountTwoSelector = await findAnotherAccountFromAccountList(
-          driver,
-          2,
-          'Account 2',
-        );
-        await driver.clickElement(accountTwoSelector);
+        await driver.clickElement('[data-testid="account-menu-icon"]');
```

Comment:
So, on top of accountListItem, I added an extra layer of Box to hide this accountListItem in the menu but to show it when the user searches for it since in the menu hidden list is behind an accordion. It was causing the test to fail so I changed the class name. It still has the same structure but rather than using the function it now tests with class. Should we revert it?